### ISSUE TITLE
How can i the change the provided classes on the mapping into my own [Token_Classification_Named_Entity_Recognition]

### ISSUE THREAD

**Issue by 181473h** [NVIDIA/NeMo#1433]
Hi, I am using the Token_Classification_Named_Entity_Recognition tutorial for a project.

By default, the dataset provided these labels: 
LOC = Geographical Entity
ORG = Organization
PER = Person
GPE = Geopolitical Entity
TIME = Time indicator
ART = Artifact
EVE = Event
NAT = Natural Phenomenon

How can I change the classes in the provided mapping such that it can use my labels and text file instead?

(My labels include: label_names = ['O', 'B-Product', 'B-Processor', 'B-WindowVersion','B-GPU','B-Display','B-MemoryStorage','B-Storage','B-Weight','B-Warranty','B-Price'] )

Attached is the error I received.
![gh2](https://user-images.githubusercontent.com/45226851/98646694-ad8ce980-236e-11eb-82e5-6ef86bf05d07.PNG)

**Comment by ekmb** [comment]
If your labels are not a subset of the provided labels, it's recommended to use [this script](https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/token_classification/token_classification.py). Note, you `shouldn't` use a pre-trained TokenClassificationModel, instead start fine-tuning with the original BERT model. When you provide a path to your data directory, the text to int labels mapping will be created automatically based on the training data.

The data directory should follow the format described in the script or in the [tutorial](https://github.com/NVIDIA/NeMo/blob/main/tutorials/nlp/Token_Classification_Named_Entity_Recognition.ipynb).

**Comment by 181473h** [comment]
Data directory format do you mean `DATA_DIR = "DATA_DIR"` this?

Because I use with my own set of data (instead of downloading preprocessed data) I get the error when I use the default pathing as shown in the attached image.

So I use `DATA_DIR = "/content"` instead
![gh3](https://user-images.githubusercontent.com/45226851/98704950-2ebc9e80-23b8-11eb-9468-46b928b62fcc.PNG)

**Comment by ekmb** [comment]
Could you please try following the `Model Training From Scratch` section of the [tutorial](https://github.com/NVIDIA/NeMo/blob/main/tutorials/nlp/Token_Classification_Named_Entity_Recognition.ipynb) and initialize your model with `model = nemo_nlp.models.TokenClassificationModel(cfg=config.model, trainer=trainer)`?
This should create labels mapping from your training data.

If you're running locally, it might be easier to use the training script with the default config: 
`python examples/nlp/token_classification/token_classification.py model.dataset.data_dir=<PATH_TO_DATA_DIR>`

If the issue persists, please add the output of the model initialization call here.

**Comment by 181473h** [comment]
The `model = nemo_nlp.models.TokenClassificationModel(cfg=config.model, trainer=trainer)` has the correct labels to the mapping (attached img 1)
![gh5](https://user-images.githubusercontent.com/45226851/98740753-88d45880-23e6-11eb-976d-e2846b629e0e.PNG)

But the last line in img 2 shows that it reverted back to using the provided labels
![gh6](https://user-images.githubusercontent.com/45226851/98740758-8b36b280-23e6-11eb-8d13-e13c9c6e30cc.PNG)
